README.md

```diff
@@ -1 +1,8 @@
 # Survey-Form-Review
+> This is Microverse Trials Project used to apply code review skills.
+
+# Acknowledgements
+I would like to give credit to list of contributors:
+
+- Family.
+- Microverse Team.
```

Comment:
I would like to commend your efforts in creating the README for the project. It's great to see that you have identified the collaborators involved, which is an important aspect. However, I would like to suggest an enhancement that could further improve the overall quality of the project.

- You can use this [README template](https://github.com/microverseinc/readme-template). It is well descriptive but kindly customize it to your project.

 - [ ] I kindly request you to consider incorporating this template into the README, taking the opportunity to provide a clear overview of the project, highlight its key features, and guide potential users on installation and usage. Additionally, including instructions on how to contribute to the project will encourage community involvement and collaboration.